packages/typespec-test/test/todo_non_branded/generated/typespec-ts/package.json

```diff
@@ -51,7 +51,7 @@
   "autoPublish": false,
   "dependencies": {
     "tslib": "^2.2.0",
-    "@typespec/ts-http-runtime": "1.0.0-alpha.20231129.4"
+    "@typespec/ts-http-runtime": "1.0.0-alpha.20240226.9"
```

Comment:
missing some code? i believe we have version included in ts files not only in package.json.